ukf_localisation:
  update_frequency: 20.0
  frame_id: 'world'
  child_frame_id: 'base_footprint'

  alpha: 1.0
  beta: 1.0
  gamma: 2.0
  omega_decay: 1.0

  # State: x, y, theta, v_x, v_y, omega
  init_state: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0]
  init_covariance: [1.0, 0.0, 0.0, 0.0, 0.0, 0.0,
                    0.0, 1.0, 0.0, 0.0, 0.0, 0.0,
                    0.0, 0.0, 1.0, 0.0, 0.0, 0.0,
                    0.0, 0.0, 0.0, 1.0, 0.0, 0.0,
                    0.0, 0.0, 0.0, 0.0, 1.0, 0.0,
                    0.0, 0.0, 0.0, 0.0, 0.0, 1.0]
  process_noise_covariance: [0.01, 0.0, 0.0, 0.0, 0.0, 0.0,
                             0.0, 0.01, 0.0, 0.0, 0.0, 0.0,
                             0.0, 0.0, 0.01, 0.0, 0.0, 0.0,
                             0.0, 0.0, 0.0, 0.01, 0.0, 0.0,
                             0.0, 0.0, 0.0, 0.0, 0.01, 0.0,
                             0.0, 0.0, 0.0, 0.0, 0.0, 0.01]
